### DISCUSSION TITLE
Feature Request: Note Sectioning and Subsectioning

### DISCUSSION THREAD

**Discussion by NamesNetworkNic** [standardnotes/forum#2807] [Suggestions]
I have been planning a TTRPG campaign recently, but because of the thoroughness of my writing style, my extremely dense, very long, arguable over organised world building notes are beginning to get out of hand, and it's becoming harder and harder to read through them without losing place.

The simplist answer to this problem would be to seperate every note into it's own tag and every section into it's own note but that would be very inconvient to read (whenever I leave one note to check something, I'd have to exit focused writing, change my selected note to the one I want to check, find what I was looking for, and then go back, spending even time getting back to where I was in the first note, then turn focused writing back on), so I came up with an idea.

Have you ever been on the mobile site for wikipedia?

It has this lovely feature where the different sections of a topic are shrunk down into open-able and close-able menus, displaying the name of their topic.

https://user-images.githubusercontent.com/68761174/173347690-1fa6e4d1-16b6-4b4e-a24b-aa750169b127.mp4

If I had that for my notes, perferably being able to subdivide even further with sections within sections, this constant inconvience would be gone for me.

And in my case it doesn't even need to be for the plaintext editor either, I have the paid version of Standard notes and have access to all of the editors, I just don't used them as I find formatting to be mostly unnecessary for anything non-public facing, but if any of the editors (preferably several) were to add this feature, it would be a significant boon towards the legibility of my notes and my productivity in general.

xo Nicolete Lovejoy

**Comment by atmoio** [comment]
I think "notebooks" might be what you're looking for. And I agree, tags aren't the best tool for every job. I think what we can do is expand the tags concept to allow for notebooks, which are essentially tags, but have a different aesthetic to them.

Tags:
- ideas
- recipes
- thoughts

Notebooks:
- Annual Report 2022
   - Introduction
   - Progress
   - Conclusion

etc.

This feature wouldn't be that hard to implement in theory, but it is practically difficult because we need to implement it in two places: web/desktop, and mobile.

This is why our goal right now is to unify our app codebases into a single one, so we don't have to build every feature twice, and we can start shipping cool new features like this without the complexity of coordinate project timelines across two different teams essentially.

Related: https://www.reddit.com/r/StandardNotes/comments/v45otg/multiple_notebooks_and_a_suggestion_on_how_to/
